### ISSUE TITLE
object literals, with named functions collide with imports

### ISSUE THREAD

**Issue by stefanpenner** [esperantojs/esperanto#122]
input:

```js
import { foo } from './foo';
var obj = {
  foo: function foo() {
    return true;
  }
};

foo();
```

output:

```js
define(['./foo'], function (___foo) {
  'use strict';
​
  var obj = {
    foo: function ___foo.foo() {
      return true;
    }
  };
});
  ___foo.foo();
```

expected output:

```js
define(['./foo'], function (___foo) {
​
  'use strict';
​
  var obj = {
    foo: function foo() {
      return true;
    }
  };
​
  ___foo.foo();
});
```

workaround:

```js
import { foo as otherFoo } from './foo';
```

**Comment by stefanpenner** [comment]
also a problem with default imports

input:
```js
import  foo from './foo';
var obj = {
  foo: function foo() {
    return true;
  }
};
foo();
```

output:
```js
define(['./foo'], function (foo) {

  'use strict';

  var obj = {
    foo: function foo['default']() {
      return true;
    }
  };
  foo['default']();

});
```

expected output:

```js
define(['./foo'], function (foo) {

  'use strict';

  var obj = {
    foo: function foo() {
      return true;
    }
  };
  foo['default']();

});
```

**Comment by Rich-Harris** [comment]
Fixed in 0.6.19. Thanks for the test case @stefanpenner

**Comment by stefanpenner** [comment]
Oh man. Thanks dude!! Will include this with the next ember cli release
